_component_design/buttons.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Buttons
+title: Simple buttons
 parent: Inputs and Controls
 layout: component
 category: Components
 usage: >
-  Buttons direct user attention to important actions that can be performed from the current page. Base the use of various types of buttons on the importance of each action.
+  A button provides a mechanism for a user to perform an action or initiate a process on a Control Panel page. A button directs the user's attention to important actions. The weight of a button (primary, secondary, or tertiary) highlight the relative importance of the action.
```

Comment:
s/A button provides a mechanism for a user to perform/Buttons allow a user to perform...